### DISCUSSION TITLE
superior_visited_island_leader placeholder

### DISCUSSION THREAD

**Discussion by kristophergeo** [BG-Software-LLC/SuperiorSkyblock2#1905] [Pending Feature Requests] [answered]
`superior_visited_island_leader` <- Returns the name of the leader from the island you are standing in/visiting

**Comment by Tuogptuan** [comment, marked as answer]
`The placeholders will show information of the island of the player. If you want to get information about the island that the player stands inside, add "location" to the placeholder: superior_island_location_<placeholder>`

**Comment by Lemvs** [comment]
@OmerBenGera you can close it.
